hummingbot/client/config/fee_overrides_config_map.py

```diff
@@ -28,4 +28,10 @@ def new_fee_config_var(key):
     "kucoin_taker_fee": new_fee_config_var("kucoin_taker_fee"),
     "kraken_maker_fee": new_fee_config_var("kraken_maker_fee"),
     "kraken_taker_fee": new_fee_config_var("kraken_taker_fee"),
+    "dolomite_maker_fee": new_fee_config_var("dolomite_maker_fee"),
```

Comment:
For DEXs, this fee is the actual amount (while it's a percent for CEXs), to avoid confusion I think we should name it `_maker_fee_amount` and `_taker_fee_amount`.

Reply:
done! Please check again.